control-plane/config/post-install/200-controller-cluster-role.yaml

```diff
@@ -20,4 +20,4 @@ metadata:
   name: knative-kafka-controller-post-install
   labels:
     app.kubernetes.io/version: devel
-rules: []
+rules: null
```

Comment:
Wondering, if we can remove the `rules` completely and that way fall back to the api servers defaults :thinking: